dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1221,6 +1221,15 @@ serverCmd()
   SAVE_OPENJ9_JAVA_OPTIONS=${OPENJ9_JAVA_OPTIONS}
   OPENJ9_JAVA_OPTIONS=${SERVER_IBM_JAVA_OPTIONS}
 
+  # Check for any verbose:gc variable set by user
+  # By default we set this to be true unless user specifies otherwise
+  # if [ ! jvmargs, javaoptions, jvmoptionsquoted contains verbose:gc, verbosegc, etc] then SERVER_IBM_JAVA_OPTIONS += -Xverbosegclog:verbosegc.%seq.log,10,1024
+  if ! [[ $JVM_OPTIONS_QUOTED == *"verbosegc"* || $JVM_OPTIONS_QUOTED == *"verbose:gc"* || ! -z "${VERBOSEGC}" ]]
```

Comment:
Though unlikely, someone might specify `VERBOSEGC=true`, so it's better to explicitly check for `false` here instead of just the existence of the envar.